src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -207,9 +220,7 @@ def upgrade():
     )
     op.add_column(
         "privacyexperienceconfighistory",
-        sa.Column(
-            "language", sa.String(), nullable=True
-        ),  # temporarily nullable for existing values
```

Comment:
no longer temporarily nullable, it's nullable. We're not going to backfill these.